src/Reader/Reader.php

```diff
@@ -315,7 +315,9 @@ public static function importString($string)
         }
 
         $libxmlErrflag = libxml_use_internal_errors(true);
-        $oldValue      = libxml_disable_entity_loader(true);
+        if (\LIBXML_VERSION < 20900) {
+            $oldValue      = libxml_disable_entity_loader(true);
+        }
```

Comment:
Please see #25 for a better solution to avoid repetitions.

Reply:
Thanks, I just saw that PR and have changed my PR to use that method switch as well 👍